capabilities: app's commands allowed by default (#3212)
tauri-apps/tauri-docs: src/content/docs/security/capabilities.mdx

}
```

{/* TODO maybe a link to an example would be better here. */}

> By default, all commands that you defined in your app
> (using the
> [`#[tauri::command]`](https://docs.rs/tauri/2.0.0/tauri/attr.command.html)
> macro)

Author: ```suggestion
By default, all commands that you registered in your app
(using the
[`tauri::Builder::.invoke_handler`](https://docs.rs/tauri/2.0.0/tauri/struct.Builder.html#method.invoke_handler)
function)
```

> are allowed to be used by all the windows and webviews of the app.

Reviewer: This should be commands registered with `tauri::Builder::invoke_handler`

Author: I implemented the suggestion, please see.

> To change that, consider using
> [`AppManifest::commands`](https://docs.rs/tauri-build/2.0.0/tauri_build/struct.AppManifest.html#method.commands).
> 

Reviewer: ```suggestion
To change that, consider using
[`AppManifest::commands`](https://docs.rs/tauri-build/2.0.0/tauri_build/struct.AppManifest.html#method.commands).

```rust title=src-tauri/build.rs
fn main() {
    tauri_build::try_build(
        tauri_build::Attributes::new()
            .app_manifest(tauri_build::AppManifest::new().commands(&["your_command"])),
    )
    .unwrap();
}
```

}
```

> {/* TODO maybe a link to an example would be better here. */}

Reviewer: Let's remove this as we have an example now

```suggestion
```

By default, all commands that you registered in your app
(using the
[`tauri::Builder::.invoke_handler`](https://docs.rs/tauri/2.0.0/tauri/struct.Builder.html#method.invoke_handler)
function)
are allowed to be used by all the windows and webviews of the app.
To change that, consider using
[`AppManifest::commands`](https://docs.rs/tauri-build/2.0.0/tauri_build/struct.AppManifest.html#method.commands).

```rust title=src-tauri/build.rs
fn main() {
    tauri_build::try_build(
        tauri_build::Attributes::new()
            .app_manifest(tauri_build::AppManifest::new().commands(&["your_command"])),
    )

>     .unwrap();
> }
> ## Target Platform

Author: ````suggestion
    .unwrap();
}
```

## Target Platform
````
